docs/README.md

```diff
@@ -8,3 +8,7 @@
 - [Package Configuration Reference](configuration.md)
 - [CI Acceleration and Caching](ci-caching.md)
 - [Design Philosophy](design-philosophy.md)
+
+# Quick Start Acceleration
```

Comment:
I'd recommend putting this at the top: before the CI and CLI section.

Reply:
For Python,  do you think we should link to pypi yourbase from here?

Reply:
Yes! 👍